### ISSUE TITLE
ComboBox help text isn't announced by TalkBack

### ISSUE THREAD

**Issue by LFDanLu** [adobe/react-spectrum#2577]
# 🐛 Bug Report 

When the user moves the TalkBack cursor onto the ComboBox input field, TalkBack doesn't announce the associated help/error text.

## 🤔 Expected Behavior

TalkBack should read out the associated help/error text if any

## 😯 Current Behavior

TalkBack doesn't mention the help text at all

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   |. 3.14
| Browser          |
| Operating System |

## 🧢 Your Company/Team

RSP
